packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -788,6 +788,7 @@ describe("Arm Template Failed Test", () => {
         properties: {
           targetResource: {
             resourceName: "test resource",
+            id: "/subscriptions/1756abc0-3554-4341-8d6a-46674962ffff/resourceGroups/test52-dev-rg/providers/Microsoft.Resources/deployments/addTeamsFxConfigurations",
```

Comment:
maybe change the subscription id and resource group name to some value that people thinks they fake data at first glance? e.g. all zero guid and fake rg group name

Reply:
It is fixed.